src/middlewared/middlewared/plugins/network_/dns.py

```diff
@@ -26,15 +55,9 @@ def query(self, filters, options):
         with contextlib.suppress(Exception):
             with open('/etc/resolv.conf') as f:
                 for line in filter(lambda x: x.startswith('nameserver'), f):
-                    ip = line[len('nameserver'):].strip()
```

Comment:
User can (and sometimes) manually edit resolve.conf which is why we were validating the ip address here.